lib/src/main/java/org/team100/lib/config/SimulatedCamera.java

```diff
@@ -0,0 +1,58 @@
+package org.team100.lib.config;
+
+import java.util.ArrayList;
+import java.util.Optional;
+
+import org.team100.lib.localization.PoseEstimationHelper;
+
+import edu.wpi.first.math.MathUtil;
+import edu.wpi.first.math.geometry.Pose2d;
+import edu.wpi.first.math.geometry.Rotation2d;
+import edu.wpi.first.math.geometry.Rotation3d;
+import edu.wpi.first.math.geometry.Transform3d;
+import edu.wpi.first.math.geometry.Translation2d;
+
+public class SimulatedCamera {
```

Comment:
i'm confused about what this class does.  it seems like it's partly the camera transformation (which could be literally one line) and it's partly some other logic.  what does this class **MEAN**?